remote-build.exp

```diff
@@ -123,6 +128,13 @@ if { ! [file exists configs/${at_version}] } {
 puts "AT version: $at_version"
 puts "Servers: $servers"
 
+# Define default value of variables if not previously defined
+# Directory used by the script to save its files.
+set_def tray_dir "~/$basedir/tmp/at-build-tray"
+# Directory where AT will be installed (aka DESTDIR).
+set_def destdir "~/$basedir/opt"
```

Comment:
I didn't mean to say "default behavior", just "behavior".
Especially if "tmp/" is removed from tray_dir.